boards/arm/mps2_an521/board.cmake

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 set(EMU_PLATFORM qemu)
+set(TFMBOARD "AN521")
```

Comment:
Can you rename to TFM_BOARD, looks nicer.
Alternatively you can use TFM's terminology : TARGET_PLATFORM

Reply:
Maybe `TFM_TARGET_PLATFORM`, though large, is the clearest?

Reply:
Yes, that's good.